src/db/write.rs

```diff
@@ -31,14 +30,18 @@ pub struct Write<'a> {
     meta: Meta,
 }
 
+#[derive(Debug)]
```

Comment:
This wasn't completely necessary but nice for consistency, and if other errors come up during further dev of InitDB.